scripts/apps/authoring/track-changes/inline-comments.js

```diff
@@ -38,19 +46,36 @@ function getCommentsFromField(customFields, resolved = true) {
     if (resolved) {
         return (obj) => ({
             fieldName: getLabelForFieldId(getFieldId(obj.contentKey), customFields),
-            comments: getCustomDataFromEditorRawState(
-                obj[fieldsMetaKeys.draftjsState],
-                editor3DataKeys.RESOLVED_COMMENTS_HISTORY
-            ) || [],
+            comments:
+        getCustomDataFromEditorRawState(
+            obj[fieldsMetaKeys.draftjsState],
+            editor3DataKeys.RESOLVED_COMMENTS_HISTORY
+        ) || [],
         });
     } else {
-        return (obj) => ({
-            fieldName: getLabelForFieldId(getFieldId(obj.contentKey), customFields),
-            comments: Object.values(getCustomDataFromEditorRawState(
-                obj[fieldsMetaKeys.draftjsState],
-                editor3DataKeys.MULTIPLE_HIGHLIGHTS
-            ).highlightsData || {}).filter((h) => h.type === 'COMMENT'),
-        });
+        return (obj) => {
+            const highlightsObject =
+        getCustomDataFromEditorRawState(
```

Comment:
That was probably a plugin I have for [prettier](https://github.com/prettier/prettier). I don't know why it would do that but both prettier and eslint consider it as "good" indentation.